packages/expo-local-authentication/src/LocalAuthentication.ts

```diff
@@ -48,9 +48,12 @@ export async function authenticateAsync(
   }
 
   if (Platform.OS === 'ios') {
+    if (!options.promptMessage) {
+      options.promptMessage = 'Authenticate';
```

Comment:
I'm not a big fan of modifying an object that was passed as an argument to the function. This may cause some unexpected behavior for the user who use that object also for something else.
It might be better to do sth like `const promptMessage = options.promptMessage || 'Authenticate';` on top of the function, below the if block with UnavailabilityError 😉 
And then call the native method in that way:
```javascript
await ExpoLocalAuthentication.authenticateAsync({
  ...options,
  promptMessage,
});
```